plottable.css

```
@@ -112,11 +112,84 @@
   fill: #32313F;
 }
 
-.plottable .bar-label-text-area text,
-.plottable .scatter-label-text-area text {
+.plottable .tick-label-container.label-12 text,
+.plottable .bar-label-text-area.label-12 text,
+.plottable .scatter-label-text-area.label-12 text {
   font-size: 12px;
 }
 
+.plottable .tick-label-container.label-13 text,
+.plottable .bar-label-text-area.label-13 text,
+.plottable .scatter-label-text-area.label-13 text {
+  font-size: 13px;
+}
+
+.plottable .tick-label-container.label-14 text,
+.plottable .bar-label-text-area.label-14 text,
+.plottable .scatter-label-text-area.label-14 text {
+  font-size: 14px;
+}
+
+.plottable .tick-label-container.label-15 text,
+.plottable .bar-label-text-area.label-15 text,
+.plottable .scatter-label-text-area.label-15 text {
+  font-size: 15px;
+}
+
+.plottable .tick-label-container.label-16 text,
+.plottable .bar-label-text-area.label-16 text,
+.plottable .scatter-label-text-area.label-16 text {
+  font-size: 16px;
+}
+
+.plottable .tick-label-container.label-17 text,
+.plottable .bar-label-text-area.label-17 text,
+.plottable .scatter-label-text-area.label-13 text {
```

This should be 17, which would probably explain your "stutter" issue
```suggestion
.plottable .scatter-label-text-area.label-17 text {
```

---

Doh 🤦